spec/models/user_spec.rb

```diff
@@ -761,29 +761,6 @@
     end
   end
 
-  describe '#age_in_years' do
-    let(:user) { create(:user) }
-
-    it 'returns an integer' do
-      expect(user.age_in_years.class).to be(Integer)
-    end
-
-    it 'counts the year if after user birthday' do
-      user.dob = Date.today.advance(days: -1, years: -10)
-      expect(user.age_in_years).to be(10)
-    end
-
-    it 'counts the year if on user birthday' do
-      user.dob = Date.today.advance(years: -10)
-      expect(user.age_in_years).to be(10)
-    end
-
-    it 'doesnt count the year if before user birthday' do
-      user.dob = Date.today.advance(years: -10, days: 1)
-      expect(user.age_in_years).to be(9)
-    end
-  end
-
```

Comment:
These very relevant test cases should be migrated to the `below_forum_age_requirement` part below -- unless they already are?
I mean, fake a user with these "edge case" birthdates and then run the `below?` method on them.

Reply:
We already have tests covering `below_forum_age_requirement` - some cases were extra here because of the implementation quirks of `age_in_years`